blog/2024-12-15-openhab-4-3-release.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: BlogPost
+category: blog
+title: openHAB 4.3 Release
+author: Florian Hotze and others
+date: '2024-12-15T18:00:00+01:00'
+previewimage: /uploads/2024-12-15-openhab-4-3-release/header.png
+tags:
+  - news
+excerpt: >-
+
+---
+
+# Activity
+
+# Highlights
+
+As you might have already noticed in the activity statistics, there was much development on openHAB - too much to list everything.
+Please refer to the [**release notes**](https://github.com/openhab/openhab-distro/releases/tag/4.3.0) to check out what's new and noteworthy (e.g. breaking changes).
+
+In the following sections, our maintainers and contributors are introducing some new features you don't want to miss!
+
+## Core Runtime Enhancements
+
+## Main UI Enhancements
+
+_Florian Hotze ([@florian-h05](https://github.com/florian-h05)), openHAB Maintainer_
+
+## Blockly Enhancements
+
+_Stefan Höhn ([@stefan-hoehn](https://github.com/stefan-hoehn)), Blockly Maintainer_
+
+## Sitemap Enhancements
+
+_Laurent Garnier ([@lolodomo](https://github.com/lolodomo)), openHAB Maintainer_
+
+## Basic UI Enhancements
+
+_Laurent Garnier ([@lolodomo](https://github.com/lolodomo)), openHAB Maintainer_
+
+## Add-on Enhancements
+
+openHAB 4.3 brings XX new add-ons, adding support for new devices:
+
+But not only were new bindings added, some existing bindings also received massive improvements:
+
```

Comment:
```suggestion

- KNX binding provides better support for KNX Secure, allowing to include keyring files and reading KNX Data Secure packets.

```

Reply:
Added in https://github.com/openhab/website/pull/499/commits/d5c892dd4a7729dc8b504364ef5f2f86361d27f0.